### PR TITLE
Pull request for given issue numbers

### PR DESCRIPTION
**Pull request by kanchannms**
Issue No.	Description
10		Create Subscription request
11		Deactivate subscription request
12		Save Inbox Call Details
16		Get Subscriber details
163		Scheduling of weekly messages
167		Rules for managing subscription state
173		OBD delivery failure statuses
174		Failed call retry rules

### CODE REVIEW

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

File: pom.xml [hunk 1]
```diff
@@ -52,6 +52,7 @@
         <spring.osgi.core.version>1.2.1</spring.osgi.core.version>
 
         <javax.persistence.version>2.1.0</javax.persistence.version>
+        <exam.version>4.2.0</exam.version>
```
Comment (koshalt) [inline pom.xml]: Why is this being added in again?

**Comment by koshalt** [comment]
@PriyaJain5 @kanchannms @AnkitGaurNms Are you actively looking into the failures from the CI builds?

File: pom.xml (hunk 1 above)
Comment (kanchannms) [inline pom.xml]: Thanks for pointing that out. We have removed it now. The changes will be visible in next pull request.

**Comment by kanchannms** [comment]
@Koshal, the CI failures are due to test cases which are failing as the actual output is different from expected output. We already have bugs in JIRA for such failures.
Some of them have been clarified by Rob and couple of them are under discussion.

Regarding the pull requests, we had earlier planned to have two branches for test case development - aricent branch and aricent reviewed branch. The pull request 281 was raised accordingly.

Later, after discussion with Rob, we decided to have only one branch here for FT cases development i.e. aricent branch. After approval, the cases shall be merged to master branch. I shall forward you the related mail. For this reason only, pull request 286 (current request) is going into master.

**Comment by frankhuster** [comment]
@kanchannms, I'm going to close & abandon this pull request as I assume the tests are now in individual pull requests. Do let me know if I should wait.

**Pull request closed without merging**